templates/theme-default/layouts/partials/components/c-module-header.html

```diff
@@ -8,29 +8,29 @@
   "context" $params
 ) . -}}
 
```

Comment:
i would completely remove this content-type  / partial as it's to specialized and it costs an additional type ;)
It also leads to more complexity for the editor because one module header would likely never be used twice.
So why not add the header fields to the modules directly

Reply:
I think the migration has already been removed ;)

Reply:
Done. The partial has now also been removed.